pyface/qt/QtOpenGL.py

```diff
@@ -16,7 +16,7 @@
     from PyQt5.QtOpenGL import *
 
 elif qt_api == "pyqt6":
-    from PyQt6.QtOpenGL import *
+    from PyQt6.QtGui import *
```

Comment:
Is this change necessary? I still see a `PyQt6.QtOpenGL` subpackage.

Reply:
That looks like an error - there may be something we need to do for Qt5 and 6 because the content of QtOpenGL is different between the two - but it can be in a different PR if needed.

Reply:
```suggestion
    from PyQt6.QtOpenGL import *
```